### PR TITLE
Implementing pixel cluster trend plots

### PR DESCRIPTION
**Pull request by leggat**
Lumisection based cluster trend plots for the barrel and endcap of the pixel detector DQM

### CODE REVIEW

File: DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc [hunk 1]
```diff
@@ -132,6 +132,20 @@ void SiPixelClusterSource::bookHistograms(DQMStore::IBooker & iBooker, edm::Run
     ss2.str(std::string()); ss2 << "Clusters -Z Disk" << i << ";Global X (cm);Global Y (cm)";
     meClPosDiskmz.push_back(iBooker.book2D(ss1.str(),ss2.str(),80,-20.,20.,80,-20.,20.));
   }
+
+  //Book trend cluster plots for barrel and endcap. Lumisections for offline and second for online - taken from strips
+  iBooker.setCurrentFolder(topFolderName_+"/Barrel");                                           
+  ss1.str(std::string()); ss1 << "totalNumberOfClustersProfile_siPixelClusters_Barrel";
+  ss2.str(std::string()); ss2 << "Total number of barrel clusters profile;Lumisection;";
+  meClusBarrelProf = iBooker.bookProfile(ss1.str(),ss2.str(),2400,0.,150,0,0,"");
+  meClusBarrelProf->getTH1()->SetCanExtend(TH1::kAllAxes);
```
Comment (dmitrijus) [inline DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc]: Is it possible not to use SetCanExtend? It does cause performance problems. And who knows what other issues in the future.

Comment (dmitrijus) [inline DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc]: We already had this discussion in 81x... sorry :(

**Comment by dmitrijus** [comment]
+1

**Comment by dmitrijus** [comment]
please test

**Comment by davidlange6** [comment]
+1

**Pull request merged**